pipeline/requirements.txt

```diff
@@ -8,3 +8,6 @@ requirements-parser==0.2.0
 # GitHub Action python does not have this default package
 datetime==4.3
 condastats==0.1.3
+pytest
```

Comment:
Minor: can you pin the versions of these libraries to whatever you're using locally? (e.g. `pytest==5.2.2`)